admin/src/pages/List.jsx

```
@@ -68,12 +71,12 @@ const List = ({ token }) => {
               {currency}
               {item.price}
             </p>
-            <p
+            <button
               onClick={() => removeProduct(item._id)}
-              className="text-center text-white w-[25%] bg-gray-700 md:text-center md:w-full cursor-pointer hover:bg-red-700 md:text-white px-4 py-2 sm:px-7 sm:py-2 rounded-full text-xs sm:text-sm"
+              className="text-center text-white w-[25%] bg-red-500 md:text-center md:w-full cursor-pointer hover:bg-red-700 md:text-white px-4 py-2 sm:px-7 sm:py-2 rounded-full text-xs sm:text-sm"
             >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)